.pipelines/vhd-builder-windows.yaml

```diff
@@ -17,13 +32,76 @@ jobs:
       -e TENANT_ID=${TENANT_ID} \
       -e AZURE_RESOURCE_GROUP_NAME=${AZURE_RESOURCE_GROUP_NAME} \
       -e AZURE_LOCATION=${AZURE_LOCATION} \
-      -e AZURE_VM_SIZE=${AZURE_VM_SIZE} \
+      -e AZURE_VM_SIZE=${PACKER_VM_SIZE} \
       -e GIT_VERSION=$(Build.SourceVersion) \
       -e BUILD_ID=$(Build.BuildId) \
       -e BUILD_NUMBER=$(Build.BuildNumber) \
       ${DEIS_GO_DEV_IMAGE} make run-packer-windows
     displayName: Building windows VHD
-  - task: PublishPipelineArtifact@0
+  - task: PublishPipelineArtifact@1
     inputs:
-      artifactName: 'vhd-release-notes'
-      targetPath: 'release-notes.txt'
+      artifact: 'vhd-release-notes'
+      path: 'release-notes.txt'
+  - script: |
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/aks-engine \
+      -w /go/src/github.com/Azure/aks-engine \
+      ${DEIS_GO_DEV_IMAGE} make bootstrap
+    displayName: build aks-engine-test - bootstrap

```

Comment:
this shouldn't be required as a separate step, `make build` depends on `make generate` which itself depends on bootstrap

Reply:
lgtm otherwise